src/ring_theory/trace.lean

```diff
@@ -232,20 +233,29 @@ end
 
 end intermediate_field.adjoin_simple
 
-lemma trace_eq_sum_roots [finite_dimensional K L]
```

Comment:
`trace_eq_sum_roots` still exists with the same statement, it's just that the first part of the proof was split off into `trace_eq_trace_adjoin` and the diff is having trouble showing that clearly.